src/config/Config.ts

```diff
@@ -118,7 +118,6 @@ const Config = {
                 UserIntegrationType.WEBHOOK
             ]),
             experimental: sortedIntegrationList([
-                UserIntegrationType.ANSIBLE,
```

Comment:
It's better to leave them here and in beta, they are not inherited.

Reply:
Aaaah, I though we had to remove them from `experimental` and `beta` and only leave them in `stable`. All right, corrected!